lib/course_planner_web/router.ex

```diff
@@ -55,6 +55,7 @@ defmodule CoursePlannerWeb.Router do
 
     resources "/users", UserController, except: [:create, :new]
     post "/notify", UserController, :notify
+    put "/resend_email/:id", UserController, :resend_email
```

Comment:
why `put`?

Reply:
Not sure, I guess because it had the user's ID at hand.